tests/unittests/test_ds_identify.py

```diff
@@ -339,11 +339,16 @@ def write_mock(data):
         ]
```

Comment:
I'm fine with enabling the config file change in a separate commit if you'd rather, but we shouldn't change `test_ds_identify.py` in adjacent commits. Could you please rebase the changes in this file into the previous commit please? Or if you want this one squashed into the last that would work too - up to you.

Also no need to add me as a co-author.

Reply:
done.